api/environments/models.py

```diff
@@ -37,6 +37,7 @@
 class Environment(LifecycleModel):
     name = models.CharField(max_length=2000)
     created_date = models.DateTimeField("DateCreated", auto_now_add=True)
+    description = models.TextField(null=True, blank=True)
```

Comment:
what size do these default to in PGres and Oracle?

Reply:
They are actual text field without any limit